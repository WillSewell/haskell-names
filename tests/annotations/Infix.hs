module Infix where

infixr 3 $$$

($$$) x y = x y
