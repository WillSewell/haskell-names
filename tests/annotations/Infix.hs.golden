$$$      at 3:10 is a global value, Infix.($$$)
$$$      at  5:2 is a value bound here
x        at  5:7 is a value bound here
y        at  5:9 is a value bound here
x        at 5:13 is a local value defined at 5:7
x        at 5:13 is a local value defined at 5:7
y        at 5:15 is a local value defined at 5:9
y        at 5:15 is a local value defined at 5:9
